wpimath/src/main/native/include/frc/controller/ArmFeedforward.h

```diff
@@ -288,7 +288,5 @@ class WPILIB_DLLEXPORT ArmFeedforward {
 };
 }  // namespace frc
 
-#ifndef NO_PROTOBUF
```

Comment:
Why was this removed (but not the `WITH_PROTOBUF` CMake option?

Reply:
Because the cmake option was only to remove google protobuf. Our copy of nanopb is modified and always included now, so it’s safe to always include it now.

Reply:
Sounds fine to me, but then shouldn't we remove the `WITH_PROTOBUF` option?

Reply:
No, because the dynamic support in glass still uses google protobuf.

Reply:
Ah, I see- That makes sense. This is good to be resolved.